zcash_proofs/src/circuit/ecc.rs

```diff
@@ -862,16 +862,15 @@ mod test {
                 cs.set("select/v'/num", bls12_381::Scalar::one());
                 assert_eq!(cs.which_is_unsatisfied().unwrap(), "select/v' computation");
                 cs.set("select/u'/num", bls12_381::Scalar::zero());
-                assert_eq!(cs.which_is_unsatisfied().unwrap(), "select/u' computation");
             } else {
                 assert_eq!(q.u.get_value().unwrap(), bls12_381::Scalar::zero());
                 assert_eq!(q.v.get_value().unwrap(), bls12_381::Scalar::one());
 
                 cs.set("select/v'/num", u0);
                 assert_eq!(cs.which_is_unsatisfied().unwrap(), "select/v' computation");
                 cs.set("select/u'/num", v0);
-                assert_eq!(cs.which_is_unsatisfied().unwrap(), "select/u' computation");
             }
+            assert_eq!(cs.which_is_unsatisfied().unwrap(), "select/u' computation");
```

Comment:
Undo this change; it makes the test misleading. I fixed this in the 1.54 bump in `main` here: 4674291fd994a488f435a763a52b407dadeed543, so we do not need to fix it in this PR.